nano/nano_node/daemon.cpp

```diff
@@ -100,23 +106,18 @@ void nano_daemon::daemon::run (std::filesystem::path const & data_path, nano::no
 																																		  : std::function<boost::optional<uint64_t> (nano::work_version const, nano::root const &, uint64_t, std::atomic<int> &)> (nullptr));
 		try
 		{
-			// This avoid a blank prompt during any node initialization delays
-			auto initialization_text = "Starting up Nano node...";
-			std::cout << initialization_text << std::endl;
-			logger.always_log (initialization_text);
+			// This avoids a blank prompt during any node initialization delays
+			logger.info (nano::log::type::daemon, "Starting up Nano node...");
```

Comment:
I think we should keep these startup messages as std::cout to make sure the initialization process is shown in the console even when configured for file logging

Reply:
I believe if someone explicitly wants to disable console output for whatever reason then we should not print anything